e2e-tests/cypress/tests/integration/channels/messaging/direct_message_spec.js

```diff
@@ -150,12 +150,12 @@ describe('Direct Message', () => {
         cy.get('#edit_textbox').type(message).type('{enter}').wait(TIMEOUTS.HALF_SEC);
 
         // # Hover on channel header
-        cy.get('#channelHeaderDescription .header-description__text').trigger('mouseover');
+        cy.get('#channelHeaderDescription .header-description__text').trigger('mouseenter');
```

Comment:
Why use `mouseenter` here instead of `mouseover`? I can never remember the difference between these two

Reply:
previously we added a handler to the div explicitly for mouseOver thats why it as using that earlier but the new one works with mouseenter see https://floating-ui.com/docs/useHover#blockpointerevents